demisto_sdk/commands/common/git_util.py

```diff
@@ -422,21 +422,30 @@ def check_if_remote_exists(self, remote):
 
         return remote in self.repo.remotes
 
-    def _handle_prev_ver(self, prev_ver):
+    def handle_prev_ver(self, prev_ver: str = ''):
         # check for sha1 in regex
         sha1_pattern = re.compile(r'\b[0-9a-f]{40}\b', flags=re.IGNORECASE)
-        if sha1_pattern.match(prev_ver):
+        if prev_ver and sha1_pattern.match(prev_ver):
             return None, prev_ver
 
-        if '/' in prev_ver:
-            remote = prev_ver.split('/')[0]
+        if prev_ver and '/' in prev_ver:
+            prev_ver_split = prev_ver.split('/')
```

Comment:
then you can remove lines 433, 435
```suggestion
            remote, branch = prev_ver.split('/', 1)
```